packages/menu-bar/test/sub-menu.test.js

```diff
@@ -465,6 +465,33 @@ describe('open on hover', () => {
     expect(subMenu.listenOn).to.equal(buttons[0]);
   });
 
+  it('should close sub-menu on mouseover on different item', async () => {
```

Comment:
Both of the new test cases pass on the current `master` branch also so they're not correctly targeting the new behavior.